sample-app/src/components/VisualAutocomplete/VisualSearchBar.tsx

```diff
@@ -94,6 +113,35 @@ export default function VisualSearchBar({
     />
   }
 
+  function renderRecentSearches() {
+    const options = recentSearches?.getRecentSearches().map(result => {
+      return {
+        value: result.query,
+        display: (
+          <div className='flex items-center'>
+            <RecentSearchIcon />
+            <div className='ml-1'>{result.query}</div>
+          </div>
+        )
+      }
+    }) ?? [];
+
+    return <DropdownSection
+      options={options}
+      optionIdPrefix='VisualSearchBar-RecentSearches'
+      onFocusChange={value => {
+        answersActions.setQuery(value);
+        updateEntityPreviews(value);
```

Comment:
For my own edification, why do we update the entity previews when navigating between recent searches?

Reply:
ah no, we don't actually need the onFocusChange function here. It's removed